LanguageFeatures/Extension-types/exhaustiveness_flow_analysis_A01_t01.dart

```diff
@@ -19,7 +19,7 @@ void main() {
   switch (i) {
     case >0:
       x = 1;
-    case ET1(value: var __):
+    case ET1 _:
```

Comment:
OK, so now we have a variable pattern rather than an object pattern. This means that we will have a variable (named `_`) whose type is `ET`. It would be a bug if that variable were treated as if it had type `int`. So we could test that be having something like `_.myETMethod()` which would then be an error.

I don't see how the current code would be able to detect that `_` has a wrong type.

Reply:
Similarly for line 35.

Reply:
Thanks!